jobs/ScheduleBackupJob.js

```diff
@@ -80,11 +80,42 @@ class ScheduleBackupJob extends BaseJob {
   }
 
   static deleteOldBackup(job, instanceDeleted) {
+    let transactionLogsDeleteBefore;
```

Comment:
Maybe rename the variable as `transactionLogsBefore` as this is the time before which you get all logs.

Reply:
Done!